MdeModulePkg/Library/DxeCapsuleLibFmp/DxeCapsuleProcessLib.c

```diff
@@ -180,8 +180,11 @@ UpdateImageProgress (
   }
 
   Status = DisplayUpdateProgress (Completion, Color);
+  if (EFI_ERROR(Status)) {

```

Comment:
Nitpick: Please consider adding a space between EFI_ERROR and the open parenthesis.

Reply:
done.

Reply:
Thanks!